.azure-pipelines/deploy-permissions-scraper.yml

```diff
@@ -0,0 +1,76 @@
+# .NET Core Function App to Windows on Azure
+# Build a .NET Core function app and deploy it to Azure as a Windows function App.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/languages/dotnet-core
+
+trigger:
+- master
+
+variables:
+  # Azure Resource Manager connection created during pipeline creation
+  azureSubscription: '984f7c25-03a1-456c-a4c0-4b97f9d61962'
+
+  # Function app name
+  functionAppName: 'permissionscraper'
+
+  # Agent VM image name
+  vmImageName: 'vs2017-win2016'
+
+  # Working Directory
+  workingDirectory: '$(System.DefaultWorkingDirectory)/PermissionsScraper/PermissionsScraper'
+
+stages:
+- stage: Build
+  displayName: Build stage
+
+  jobs:
+  - job: Build
+    displayName: Build
+    pool:
+      vmImage: $(vmImageName)
+
+    steps:
+    - task: DotNetCoreCLI@2
+      displayName: Build
+      inputs:
+        command: 'build'
+        projects: |
+          $(workingDirectory)/*.csproj
+        arguments: --output $(System.DefaultWorkingDirectory)/publish_output --configuration Release
+
+    - task: ArchiveFiles@2
+      displayName: 'Archive files'
+      inputs:
+        rootFolderOrFile: '$(System.DefaultWorkingDirectory)/publish_output'
+        includeRootFolder: false
+        archiveType: zip
+        archiveFile: $(Build.ArtifactStagingDirectory)/$(Build.BuildId).zip
+        replaceExistingArchive: true
+
+    - publish: $(Build.ArtifactStagingDirectory)/$(Build.BuildId).zip
+      artifact: drop
+
```

Comment:
We should add a notification for when this pipelines fails when run automatically. We may need to setup this task in this ADO project.

Reply:
```yaml
  - task: YodLabs.O365PostMessage.O365PostMessageBuild.O365PostMessageBuild@0
    displayName: 'Graph Client Tooling pipeline fail notification'
    inputs:
      addressType: serviceEndpoint
      serviceEndpointName: 'microsoftgraph pipeline status'
      title: '$(Build.DefinitionName) failure notification'
      text: 'This pipeline has failed. View the build details for further information. This is a blocking failure. '
    condition: and(failed(), ne(variables['Build.Reason'], 'Manual'))
    enabled: true
```

Reply:
Do we add this once we've already setup the task in ADO?

Reply:
I've requested this task from DeVere. Let me see if we can get this added today.